src/AssemblyAI/Lemur/Requests/LemurActionItemsParams.cs

```diff
@@ -12,5 +15,46 @@ public record LemurActionItemsParams
     ///
     /// </summary>
     [JsonPropertyName("answer_format")]
-    public string? AnswerFormat { get; init; }
+    public string? AnswerFormat { get; set; }
+
+    /// <summary>
+    /// A list of completed transcripts with text. Up to a maximum of 100 files or 100 hours, whichever is lower.
+    /// Use either transcript_ids or input_text as input into LeMUR.
+    /// </summary>
+    [JsonPropertyName("transcript_ids")]
+    public IEnumerable<string>? TranscriptIds { get; set; }
+
+    /// <summary>
+    /// Custom formatted transcript data. Maximum size is the context limit of the selected model, which defaults to 100000.
+    /// Use either transcript_ids or input_text as input into LeMUR.
+    /// </summary>
+    [JsonPropertyName("input_text")]
+    public string? InputText { get; set; }
+
+    /// <summary>
+    /// Context to provide the model. This can be a string or a free-form JSON value.
+    /// </summary>
+    [JsonPropertyName("context")]
+    [JsonConverter(typeof(OneOfSerializer<OneOf<string, Dictionary<string, object?>>>))]
+    public OneOf<string, Dictionary<string, object?>>? Context { get; set; }
```

Comment:
We do the same thing [in java](https://github.com/AssemblyAI/assemblyai-java-sdk/blob/main/src/main/java/com/assemblyai/api/resources/lemur/types/LemurBaseParamsContext.java#L67) for you 

Happy to talk through possible workarounds, but we consider `map<string, object>` to be a helpful type for `object`, since a json object is of course a map of strings to values

Reply:
This is only used as a parameter, it isn't returned anywhere.
Hypothetically, if it were returned, it'd be best returned as a string, or have a method overload to specify a type to convert it to using our JSON serializer.

Reply:
Oh my mistake 😅 going to look into making these changes

Reply:
Right now, since we don't differentiate between requests or responses when generating types, so they will all be given the same typings (given the same type definition).

So, if we made this change, how would you feel if OpenAPI `object` type responses came through as `object` rather than `Dictionary<string, object?>`? That seems possibly harder to work with?

Reply:
For the request, we can do `object` which you can assign a string to or anything else.
`OneOf<string, object>` is fine too.